### ISSUE TITLE
`AggregatedTranscript()` signature incorrect in `.pyi`

### ISSUE THREAD

**Issue by fjarri** [nucypher/ferveo#129]
`ferveo/__init__.pyi` has:

```
class AggregatedTranscript:

    def __init__(self, messages: Sequence[Tuple[Validator, Transcript]]):
        ...
```

But `AggregatedTranscript::new` is defined in the bindings as 

```
#[pymethods]
impl AggregatedTranscript {
    #[new]
    pub fn new(messages: Vec<ValidatorMessage>) -> Self {
```

where `ValidatorMessage` is not exported. 

Same problem with `AffregatedTranscript.verify()` and `Dkg.aggregate_transcripts()`. This is also repeated in the copy of the type stubs in `nucypher-core`.

**Comment by piotr-roslaniec** [comment]
I don't think it's possible to a tuple definition from Rust `pyo3`:
```
#[derive(FromPyObject, Clone)]
pub struct ValidatorMessage(Validator, Transcript);
```
I will replace `ValidatorMessage` with a `pyclass`, which is a breaking change fo `nucypher`.

**Comment by piotr-roslaniec** [comment]
Closed by #131
